app/mailers/bid_ask_mailer.rb

```diff
@@ -1,9 +1,19 @@
 class BidAskMailer < ApplicationMailer
   default :from => 'noreply.alertwalutowy@gmail.com'
 
-  def send_bid_ask_email(user)
+  def send_bid_email(user,currency_name,currency_bid)
     @user = user
+    @cname = currency_name
+    @cbid = currency_bid
     mail( :to => @user.email,
-    :subject => 'Currency reached selected value')
+    :subject => 'Currency reached selected value!')
```

Comment:
```suggestion
    to: @user.email,
    subject: 'Currency reached selected value!')
```
Warto już nie używać tych "rakiet" bo to ze starych railsów praktyka